src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -55,7 +55,7 @@
 # pylint: disable=redefined-outer-name,too-many-try-statements,import-outside-toplevel,unnecessary-lambda,too-complex,invalid-name
 
 logger = logging.getLogger(__name__)
-JOBS_PURGE_TIMEOUT = timedelta(days=1)
+JOBS_PURGE_TIMEOUT = timedelta(hours=1)
```

Comment:
```suggestion
JOBS_PURGE_TIMEOUT = timedelta(hours=1, minutes=15)
```
to protect from jobs started at minute 59 of the hour.